### ISSUE TITLE
npm distribution is missing cdn files since 2.1.4

### ISSUE THREAD

**Issue by svenackermann** [jquery/jquery#2244]
Comparing `npm install jquery@2.1.3` against `npm install jquery@2.1.4` reveals that the latter does not include a folder "dist/cdn" anymore.
The main problem I have with this is that the file "dist/jquery.min.js" includes a sourcemap comment while "dist/cdn/jquery-2.1.3.min.js" did not. I cannot include the 2.1.4 file in my build without getting 404 errors int the browser (when it tries to load the sourcemap).
This might be related to #1707.

**Comment by timmywil** [comment]
Thank you for opening an issue. The inconvenience is regrettable, but there were two issues.

1. The fact that `dist/cdn` was included in 2.1.3/1.11.2 was a mistake. Those files are not meant for npm and bower and we're sorry for the confusion. We fixed this in 2.1.4.
2. We do agree that the source map URL should not be there. However, that was an issue that has been fixed for 3.0.0.

Again, I'm now wishing I took out that source map URL for the 2.1.4 release, but we all missed it. Regardless of my regret, all future releases will not include the source map URL.
